### DISCUSSION TITLE
2023-09-08 - Do you enable external bookings? - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by lukecookssw** [SSWConsulting/SSW.Rules.Content#6627] [Chewing the Fat]
Hey booking specialists!

Let's talk about [this rule](https://www.ssw.com.au/rules/meeting-bookings/)

**Comment by PennyWalker** [comment]
Having used tools like Calendly in the past, it’s a great way to avoid calendar clashes. It’s a bit trickier to reschedule appointment than via your calendar, but it’s been great for recruitment and I love that it’s free. Calendly cost $ after one link.

**Comment by matt-goldman** [comment]
Such a game changer rather than going back and forth.  It's much better than a third party service, even those that integrate with your calendar can't compete with something that's (almost) built in.

**Comment by JackDuan123** [comment]
Great to use,
can we add them to tiktok?
facebook is out-of-date

**Reply by AlexRothwell** [reply]
Good point! We should definitely have an external facing linktree or equivalent for places like tiktok. That link would be perfect there

**Reply by camillars** [reply]
❌ We don't have enough followers to be able to add a link to TikTok yet :( 
✅ BUT, we already use Linktree for social media

**Comment by tiagov8** [comment]
It was a way better experience than a normal form for this booking purpose. Of course, the standard form takes care about all other inquiries so it is still necessary.

--

My list of improvements:

1. Change title from "SSW Initial Meetings" to "Book a Free Initial Meeting with SSW"

2. I'd rather have the **city** names instead of **provinces**. In that case, include Newcastle, NSW

3. Also for "France" instead of "Francais" (unsure what's in China :D )

4. Change the sentence from

> "Book a free initial meeting with a Melbourne-based account manager and a developer to go over what SSW might be able to do for you and your business 😀"

to

> "Meet with a Melbourne-based account manager/developer to see how SSW can help you 😀"

**Reply by Geordie88** [reply]
On point 1. I think we can drop the SSW from the initial meeting as our logo is directly above it.

**Reply by SofieHong** [reply]
Agree with point 1 👍

**Comment by bradystroud** [comment]
I wonder if this can be DDOSed

**Reply by ozdemir-mehmet** [reply]
Probably

**Comment by ozdemir-mehmet** [comment]
defo no direct bookings of dev resources, it should always go thru the account manager.

**Reply by leahy268** [reply]
It would be very hard to organize our Calendars if clients could book us directly.  As an example, we could be trying to internally find a time to book  in a spec review for a client after their initial meeting with 2 Devs and have a client book one of them out before we could finalize it.

**Comment by 2fernandez** [comment]
I love this idea. 
There are a few improvements that can be made.

1. Form length - Reduce the overall length of a form;
2. Form title font - Having the title all in caps is not great; I have not seen this done in many places at SSW
2. SELECT A SERVICE - should be "Choose your location"
3. DATE / TIME - Should not be revealed unless a selection is made
4. PROVIDE ADDITIONAL INFORMATION: It should just be a textarea / text field under "ADD YOUR DETAILS"

5. The UI - As a tech company, I wish the AI handled the entire process. 

**For example:**
🤖Sophie - Hey there, how can I help you?
👨‍🦲You - Can I make a booking for an initial meeting?
🤖Sophie - Yes, what day and time would you like to make the booking?
👨‍🦲You - Wednesday, please.
🤖Sophie - What location would you like it for?
👨‍🦲You - Sydney, please.
🤖Sophie - Great, what's your contact address?
👨‍🦲You - Timbuctoo


Once we are in this format, we are not far from "Voice Actionable Commands." 

AI Layering - The above is the perfect example of AI Layering. 
AI Layering - The system is organically integrated into an environment.

❤️ AI is the new UI

**Reply by samsmithssw** [reply]
this is a really cool idea, get GPT to work with the client to find a booking - a much cooler smoother experience that keeps you on the same page

**Comment by SamBWagner** [comment]
I really liked it. The simple UI was a marked improvement over the consulting page one. If the admin on our side is nicer then it's a go for me!

**Comment by tombui99** [comment]
This is a great tool and it should 100% be used internally as well to book certain people like Adam...

**Reply by danielmackay** [reply]
Couldn't you use the built in Scheduling Assistant for that?

**Comment by danielmackay** [comment]
I see the being mentioned as tool for both internal and external use.  I think this has some benefits for external use, but believe it's terrible for internal use.

When booking internally, you can use the Scheduling Assistant.  This gives you full visibility of everyone's calendars.  What more do you need?  I would also be interested to know if Microsoft Bookings honoured your existing calendar availability.  I'd hate to have to manage two availability schedules.

When booking meetings with external customers there are two other options to consider:

1. You can share your calendar with external parties.  That way they have full visibility of your schedule, and it avoids "timeslot ping-pong".
2. You can also use "Scheduling Poll" within Outlook to get a large audience to vote on preferable timeslots.

**Comment by JeanThirion** [comment]
Booking for sales is awesome; it makes a lot of stuff easier and secure leads, I love it.

Allowing clients to book developers directly sounds like hell to me. It'd be super hard to manage availabilities, and we (ACC managers) would really loose that special relationship we have with clients. 

I think it'd go as far as losing some of our competitive advantage

**Comment by apforsyth** [comment]
love this!

**Comment by UlyssesMaclaren** [comment]
Saved me heaps of time, but I do miss having the email thread in the appointment history

**Comment by MarcusIrmscherSSW** [comment]
This is a great system, far simpler than the old method. ✅

**Comment by tkapa** [comment]
Good UX for the MS Side.

I noticed that Sophie cached my conversation from last week, I'm not sure how I feel about that.

Internal bookings could be useful. Don't expose resources to clients to book out for themselves.

**Comment by ncn-ssw** [comment]
I like that it provides the booker with immediate clarity as to when they can meet with someone. When you are waiting for a stranger to get back in touch with you, you always wonder if they are going to forget.

**Comment by jakebayliss** [comment]
I think I prefer the old form where the user can pick where they are located rather than giving them 6 different options for the same thing which fill up the entire screen. Other than that, looks great for external bookings

**Reply by 2fernandez** [reply]
Yup as a potential touch point for sales/marketing, having a long form is not ideal and can act as a deterrent.

**Comment by MattParkerDev** [comment]
Very nice looking UI and seems like a much nicer experience for clients to book initial meetings! I don’t think clients should be able to book devs directly, and I’m not sure it’s applicable internally when you have access to each others calendars :)

**Comment by joshbermanssw** [comment]
Have seen the popularity of this skyrocket recently... makes booking appts so much quicker.

**Comment by camillars** [comment]
Great tool and I love that it integrates with our calendar.

❌ Form is a bit too long - may reduce conversion rate (for leads)
✅ Saves us time and makes it easier for the client to book in time with the Account Managers

**Comment by Levijj22** [comment]
I really like these booking systems, but I worry that it makes us look lazy to a potential client. 
We've all received those sales emails, and when I read them, I always think "They can't even be bothered to find a time that works for me". 
SSW is a premium service. Although there are plenty of use cases (recruitment, booking in Sysadmins, etc.), I want to personally work with the client to find a time that works, not throw them a link to work it out themselves. 
But I'm open to someone changing my mind.

**Reply by AlexRothwell** [reply]
I can see your point here. Some people may view a tool like this as lazy but others may see it as effective scheduling which helps them get to their goal quicker.
I think it'd be good as just one of the tools you can use. Maybe for those kinds of conversations it could be included at the bottom of the message or in your signature but you still continue with the conversation if they'd like

**Reply by 2fernandez** [reply]
Levi, would it help if the AI Sophi actually helped make the booking?

eg:

🤖Sophie - Hey there, how can I help you?
👨‍🦲You - Can I make a booking for an initial meeting?
🤖Sophie – Let's make the booking online. What is your email address?
👨‍🦲You – yes, it is josephfernandez@ssw.com.au
🤖Sophie – Thanks Joseph. Does Monday the 12th at 2 pm work for you?
👨‍🦲You – Not really, but I am free on the 13th at 3 pm.
🤖Sophie – Awesome, I’ve made a booking for the 13th at 3 pm 
🤖Sophie – You should get an email notification about this shortly, or you can click here to add it to your calendar


Advantages of doing this: 
✅ UX - Reduce overall user clicks
✅ CX – Concierge AI service
✅ Branding – Give the impression we are on the bleeding edge of technology

**Reply by jakebayliss** [reply]
Agree - We dont want to lose the personal touch with our clients and I this whole process is a bit impersonal. If I was a client I would want to speak to a someone on the phone where I can sort out the meeting time, rather than a chatbot. Yes, may not be feasible if its out of office hours, but a call should be the first option given imo

**Comment by cvionnet** [comment]
✅ Easy to use
✅ Can be useful internally
❌ Allowing clients to directly book developers

**Comment by landonmaxwell** [comment]
Way better than an extended game of email tag trying to find a mutually workable time ⚡

**Comment by jomelsontanco** [comment]
✅ I like that I can book meetings now
❌ Doesn't set expectations of the next events that will happen once the book button is clicked
❓How to handle reschedules?
❓Can Sophiebot help me book meetings?

**Comment by bahjatalaadelssw** [comment]
Much like booking on a medical centre app. It is good idea

**Comment by vladislav-kir** [comment]
I like this new system. I think using this for internal bookings could be really great!

**Comment by tomek-i** [comment]
Internally:
Can someone explain to me why Microsoft Booking would make a difference in booking time with Adam or Uly compared to sending a meeting request through Outlook?


For Externally:
Yes, it makes sense to expose your schedule to the client so they can pick their date and time hassle-free.

**Comment by christoment** [comment]
The new MS booking system is so much better and personally I prefer this one.
As a customer, I feel like I have more control over the schedule instead of just bumping around and wait for the company to call me.

**Comment by Jord-Gui** [comment]
Very nice!!

**Comment by Marxoz** [comment]
I like the Microsoft bookings option, and it makes it way easier to book initial meetings. 

I won't use it to book developers directly though. Clients should always work through Account Managers

It makes sense to be able to book teams like SysAdmins and SSWTV. Even certain services like taking photos internally, reviews etc

**Comment by JackDevAU** [comment]
✅External - Allows clients to easily book an initial meeting 
✅Internal - I personally think this is a great idea for internal use. As its a quicker way to book meetings. However, ⚠️There could be the potential that this is redundant and is just another tool that isn't used in favor over a message/meeting email.

**Comment by leehawkinsssw** [comment]
A good idea for organizing meetings with external parties, but not necessary (or even desirable) for internal use.

**Comment by jeoffreyfischer** [comment]
That will definitely save some time!

**Comment by KahaMason** [comment]
Looks great!

Great idea to get the ball rolling on initial meetings and specifying availability before hand. A potential improvement to the Microsoft booking form could be to include a brief description or profile on who the client may be getting into contact with from their selected city. I wasn't sure who I would be contacting from the initial form.

**Comment by Calvin-Lieu** [comment]
Like most of the devs I agree that it wouldn't make much sense to be able to book developers directly, but this seems like a great way to improve booking efficiency in general.

**Comment by BenDunkerleySSW** [comment]
✅ As long as there are no conflicts between booking systems I think this is a good change.

**Comment by GordonBeeming** [comment]
I love this addition, this should save customers and account managers time and lower the friction barrier to getting started

There's so much less friction in booking the spot yourself (as a customer). Lots of providers send this type of link out to book demos and initial meetings of their own, and it works great.

I think there is value in having an account manager buffer, largely to avoid customers booking a developer when their needs actually need a different developer.

**Comment by Ben0189** [comment]
From Client perspective, Microsoft booking is a great tools for them. 

✅ can include individuals that client want to meet with
✅ they can select a date and time that aligns with their availability

**Comment by AshAnil** [comment]
Microsoft Bookings is a great tool for booking people internally and externals can book

**Comment by lukecookssw** [comment]
Given SSW's preference to book developers in pairs, I can't see this being a feasible solution for booking developers directly. Additionally, SSW Account Managers do a great job of trying to book devs on jobs they *want* to do. This would be lost if clients could book devs directly.

**Comment by AttackOnMorty** [comment]
Power within the ecosystem.

**Comment by jernejk** [comment]
I used it for nearly 2 years and I've seen others using it as well.
It works really well, but the links are terrible.

MS Bookings link: https://outlook.office.com/bookwithme/user/49b9d94f8231402e91afaee73c3a86c2@ssw.com.au?anonymous&ep=plink
YouCanBookMe: https://jkssw.youcanbook.me/

**Comment by stephanfakossw** [comment]
MS bookings does not work for me...if I hit refresh, it leads me to my browser Outlook.
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/96279310/70301d15-307f-47b0-9238-5e8ecbecc413)

**Reply by AntPolkanov** [reply]
Seth and I are having the same issue.

**Comment by amankumarrr** [comment]
I like the idea but I would love to embed it as an iframe on our website so that our client remains on it.

**Comment by chrisschultzssw** [comment]
It's a great solution for Initial Meetings.

I understand why the Microsoft video has Facebook in their example - targeting smaller businesses with Facebook pages - but LinkedIn seems like a better place to use it (like the example in the rule).

**Comment by AntPolkanov** [comment]
Cannot open MS Booking :( 
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/123032112/104349b2-6f22-45fe-8c91-240aa74c4472)

**Comment by Dhruv-0987** [comment]
I think the booking system needs to be streamlined and Microsoft forms are the first step to doing that, like everything other thing in our organisation booking client meeting should also follow a standard. Although The current UI for the Microsoft from is not very appealing could be better.

**Comment by jimmidier** [comment]
This new method feels a lot more standardized to the external comparing to the old way. I only wish the time selector has regard to existing Outlook calendar otherwise it's pointless.

**Comment by CoffeeBugHub** [comment]
As long as we are not relying solely on email and making the whole process easier for clients or internal employees, it is a good call.

**Comment by zacharykeeping** [comment]
This seems like a clean, streamlined solution to booking these meetings.

**Comment by Hona** [comment]
I like the bookings page. All the booking buttons should use the page not the form

**Comment by Aibono1225** [comment]
I like it, it seems that it can make the whole booking process easier.

**Comment by sethdaily** [comment]
+1 for the automatic bookings form - as a customer I much prefer seeing available times right when I'm looking for them.

**Comment by jerryluossw** [comment]
I think it's great way to Initial Meetings, but it's not a good idea to book devs directly.

**Comment by hveraus** [comment]
Overall, Microsoft Booking is a convenient way to manage their appointments and provide better customer service. I have not seen the backend yet, is that also convenient for businesses to manage availabilities?

**Comment by ozair-ashfaque** [comment]
Microsoft Booking makes it easier to book initial meetings

**Comment by SofieHong** [comment]
✅ Microsoft Booking is really useful when it comes to sales, mainly because of its UX that gives clients more options and details. One of the coolest features is the ability to choose the meeting time. 

❌ But if we're talking about using it for internal work, I don't think it's the right tool. Like Daniel mentioned, we already have a calendar to check everyone's schedule. Plus, it's important to communicate with people before sending appointments.

**Comment by KristenHu** [comment]
Really like the UX, it's simple and efficient, but there are still some translation mistakes in Chinese. I'm not sure if the French version also has the same issue.

**Comment by johnxu16** [comment]
Microsoft Booking looks far more good than I thought. It is very cohesive with other microsoft software, so with SSW products.

**Comment by sylhuang** [comment]
At first I was like, why another booking system? There are so many ready-to-use options out there. Then I see that it's integrated with outlook canlendar!!!

**Comment by babakamyljanovssw** [comment]
✅ I like the new way of booking, it saves a lot of time
❓Addition Information fields is optional, so client can book a meetings without specifying any reason?

**Comment by Berber-Mama** [comment]
The best thing about the new booking is we can easily lock in an initial meeting💡

**Comment by ricksu978** [comment]
✅Microsoft Booking ensures no overlapping booking, where Outlook Appointment/Meetings can be overlapped (looking at Adam's calendar ... 😱)

**Comment by AnkyChe** [comment]
Microsoft Booking looks nice, I love the UX it has both Chinese and French😀.

**Comment by PothieuG** [comment]
Great tool ! I especially appreciate the fact that it's integrated into the Office 365 suite.

**Comment by evecogan** [comment]
Great rule and tool! Hope the cyber security is tight because I always feel a bit nervous giving access to my calendar.

**Comment by jackreimers** [comment]
Making it easier for clients to see who is available is a great idea but I don't think they should be booking devs directly.

I think using a booking tool for internal tasks such as replacing hardware is overkill when you can just raise a ticket.

**Comment by Harry-Ross** [comment]
I think the UX is really fantastic when using Microsoft Bookings, although I was concerned by what determined which times are booked out. Is it linked to a specific person's calendar? 

Internally, I don't think this is the right tool as we already have access to others' calendars and may even be better to directly communicate with the person instead of adding an extra step to book them.

**Comment by tino-liu** [comment]
This is a great tool for external use, UI and UX is great!
It would be somewhat confusing and make things more rigid if we use this booking system internally.

**Comment by BrookJeynes** [comment]
Bot gave me the correct link and the form looked great.
